### PR TITLE
Cases table is complete with real data

### PR DESCRIPTION
**Pull request by hanseljt77**
All of the stuff I did to grab data from the backend was meant to be done in my branch "populate-cases" but I forgot to actually switch what branch I was in. Thus, this branch will close issues 4 and 5. Sorry, everyone! Should I just delete the forgotten branch?

### CODE REVIEW

**Comment by DRSisco** [comment]
Looks like you didn't pull master after Noah's merge. You will need to either merge in master to your branch OR rebase your branch on master in order to resolve those conflicts.

**Comment by DRSisco** [comment]
>All of the stuff I did to grab data from the backend was meant to be done in my branch "populate-cases" but I forgot to actually switch what branch I was in. Thus, this branch will close issues 4 and 5. Sorry, everyone! Should I just delete the forgotten branch?

You can do git cherry-pick from this branch to the other one, then remove the commits from this branch afterwards. Though that requires more git knowledge than you may feel comfortable with, seeing as you did a merge with a PR instead of the command line.
You can just reference both issues in the PR description with:
`closes #4`
`closes #5`
Obviously without the code escaping I did in the markdown here, so you actually link and reference them. That way they will close the issues when merge as well as tell your reviewer what issues they need to review your code for.

File: src/app/app-routing.module.ts [hunk 1]
```diff
@@ -4,13 +4,13 @@ import { UserlistComponent } from './userlist/userlist.component';
...
 import { AssignedCasesComponent } from './AssignedCases/AssignedCases.component';
-
+import { config } from 'rxjs';
 
 const routes: Routes = [
   { path: '', redirectTo: '/dashboard', pathMatch: 'full' },
   { path: 'dashboard', component: DashboardComponent },
   { path: 'userlist', component: UserlistComponent },
-  { path: 'case', component: CaseComponent },
+  { path: 'positive-case', component: CaseComponent },
```
Comment (noahthomas88) [inline src/app/app-routing.module.ts:13]: Do we want to change the case component to the PositiveCaseComponent? I think that name is more descriptive of what it is.

**Review by noahthomas88 — APPROVED** [review]
Great work! Is the Fix to the data not displaying in another PR? Also I have a comment about renaming the case component to PositiveCaseComponent, curious to see what you think about that change.

**Pull request closed without merging**